sandy-city-utah/targetsolutions-user-sync
# feat(packages): add/update references to packages

### What Changed?
  - add npmrc configuration for packages
  - install and update import paths to packages
  - update readme to reflect changes

### Issues Addressed:
  - P21312-19

### Issue Testing:
  - Manual Developer Testing

### Test Results:
  1. Ensure you're logged in to the GitHub package registry
  1. Run install
  1. See that it installs correctly
  1. Run the process
  1. See that it continues to run as expected

### Additional Information:
Requires authentication with GitHub packages.  Details explained in readme.
